services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/AccessTokenResponse.java

```diff
@@ -35,7 +35,7 @@ public ProvidedUserInfo parseIdToken() {
 				
 		try {
 			
-			String idTokenClaims = new String(Base64.getDecoder().decode(idTokenParts[1]), StandardCharsets.UTF_8);
+			String idTokenClaims = new String(Base64.getUrlDecoder().decode(idTokenParts[1]), StandardCharsets.UTF_8);
```

Comment:
At line 57 can you log the malformed id_token?  In the unlikely event that your fix does not address all occurrences this will help us further debug the issue.